### PR TITLE
Publish the .d.ts from our previous quickstart repo

### PR DESCRIPTION
**Pull request by alexeagle**

### CODE REVIEW

File: angular2/angular2.d.ts [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+interface List<T> extends Array<T> {
+}
+interface Type {}
+
+declare module "angular2/angular2" {
+  function bootstrap(appComponentType: any): void;
+}
+declare module "decorators" {
+  function Component({
+    /** Defines which tags in the template will be replaced with an instance of this component. */
```
Comment (rkirov) [inline angular2/angular2.d.ts]: do we intent to leave the comments in the .d.ts?

**Comment by rkirov** [comment]
LGTM.

File: angular2/angular2.d.ts (hunk 1 above)
Comment (alexeagle) [inline angular2/angular2.d.ts]: yes! if the comments appear, then IDEs show the docs in the completion hover which is really nice.

But - did we really intend to have a "decorators" module? that doesn't look right

Comment (rkirov) [inline angular2/angular2.d.ts]: Nope that is leftover, decorators are part of angular2 now.

**Pull request merged**